app/containers/ServerItem/styles.ts

```diff
@@ -7,20 +7,22 @@ export const ROW_HEIGHT = 56;
 export default StyleSheet.create({
 	serverItemContainer: {
 		flexDirection: 'row',
-		alignItems: 'center'
+		alignItems: 'center',
+		paddingHorizontal: 15,
+		paddingVertical: 12
 	},
 	serverIcon: {
 		width: 44,
 		height: 44,
-		margin: 12,
```

Comment:
You've removed a `margin`, which is top, right, bottom, left and replaced with `paddingLeft` (left) on the text container.